fastlane_core/lib/fastlane_core/core_ext/shellwords.rb

```diff
@@ -0,0 +1,60 @@
+require_relative '../helper'
+require 'shellwords'
+
+# Here be monkey patches
+
+class String
+  # CrossplatformShellwords
+  def shellescape
+    CrossplatformShellwords.shellescape(self)
+  end
+end
+
+class Array
+  def shelljoin
```

Comment:
Same question here - Does this 100% override the method defined by `shellwords` because this is required after `shellwords`?